### ISSUE TITLE
Export files to Omeka S

### ISSUE THREAD

**Issue by pprw** [tropy/tropy-plugin-omeka#2]
In my installation, this plugin does only export metadata to Omeka S, not the photo's files.

Is it the expected behavior? 

Let's say I export 300+ photos metadata to Omeka S, how can I quickly upload the file for each item?

**Comment by pdpotter** [comment]
This is not expected behavior. It can be caused by ImageMagick not being installed on the server where Omeka S is running.

See the discussion at https://forums.tropy.org/t/export-to-omeka-s-not-working/813/7.

**Comment by tortugamarina** [comment]
The plugin is somehow unstable. When uploading a record with a pdf file, it copies the complete pdf in omeka for each page, so that if the pdf had 100 pages, it will get copied 100 times. I am using linux mint and omeka -s 2.1.1   Is this a version problem, or something you're working on?

**Comment by inukshuk** [comment]
Right, at the moment the plugin uploads the original file for each photo, for PDFs that's not very useful as the the original file for each page is the full PDF. I guess the easiest solution would be to upload the image variant that Tropy renders instead of the original? In that case the pages would get exported as images.
